src/color-slider/color-slider.js

```diff
@@ -130,6 +131,13 @@ const Self = class ColorSlider extends NudeElement {
 				this._el.spinner.style.setProperty("--value-length", valueStr.length);
 			}
 		}
+		else if (name === "tooltip") {
+			if (change.value === "progress") {
```

Comment:
Don’t we still want to set these (to different values) even if the value is not progress?

Reply:
Yes, we do. I also thought about it today. I will fix it. Thanks.

Reply:
Done!